uhdm-plugin/UhdmAst.cc

```diff
@@ -1308,6 +1308,10 @@ void UhdmAst::process_initial()
                 block_node->children.push_back(node);
                 node = block_node;
             }
+            auto nested_block_node = make_ast_node(AST::AST_BLOCK);
+            nested_block_node->children.push_back(node);
+            node = nested_block_node;
+
```

Comment:
I'm unsure about this change. Does it fix some case? It is true we should replicate yosys behavior, but I think, not always there are 2x AST_BLOCK under AST_INITIAL, example case: https://github.com/antmicro/yosys/blob/uhdm-plugin/frontends/verilog/verilog_parser.y#L1993

Here we have AST_INITIAL + AST_BLOCK + AST_ASSIGN_LE

Reply:
In my case there were two AST_BLOCKs

```
module dut();
  reg a, b, c;
  initial begin
    a = 0;
    b = 1;
    c = a === b;
    c = a !== b;
  end
endmodule
```
Maybe this change doesn't help so much. I'll remove it.